src/test/regress/expected/multi_utilities.out

```diff
@@ -446,7 +446,32 @@ SELECT 1 FROM citus_create_restore_point('regression-test');
         1
 (1 row)
 
--- TODO: support VERBOSE
--- VACUUM VERBOSE dustbunnies;
--- VACUUM (FULL, VERBOSE) dustbunnies;
--- ANALYZE VERBOSE dustbunnies;
+-- verify VERBOSE is supported
+VACUUM (FULL, VERBOSE) dustbunnies;
+INFO:  vacuuming "public.dustbunnies"
+INFO:  "dustbunnies": found 0 removable, 0 nonremovable row versions in 0 pages
+DETAIL:  0 dead row versions cannot be removed yet.
+CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s.
+INFO:  FROM localhost:57638 MESSAGE: INFO:  vacuuming "public.dustbunnies_990002"
+
+INFO:  FROM localhost:57638 MESSAGE: INFO:  "dustbunnies_990002": found 0 removable, 6 nonremovable row versions in 1 pages
+DETAIL:  0 dead row versions cannot be removed yet.
+CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s.
```

Comment:
This'll probably break travis a lot. I think VERBOSE is probably not something for which we can really add a regression test.

Reply:
Than I propose that we should add (VACUUM | ANALYZE) VERBOSE to our testing document consistently. Maybe auto-testing tool can handle this?